adapters/p2p2core/block.go

```diff
@@ -34,14 +34,16 @@ func AdaptBlockHeader(h *spec.SignedBlockHeader, eventsBloom *bloom.BloomFilter)
 		Timestamp:        h.Time,
 		ProtocolVersion:  h.ProtocolVersion,
 		EventsBloom:      eventsBloom,
+		L1GasPriceETH:    AdaptUint128(h.GasPriceWei),
+		L2GasPriceETH:    &felt.Zero, // TODO: Fix when we have l2 gas price
```

Comment:
Do we really want to use `felt.Zero` for L2 gas prices even though we do not support them yet? I would prefer `nil` so that the JSON output is clearly `null` and shows that this field is always empty. Using `felt.Zero` may suggest that this field is a "valid" one and may be misleading.